diff --git a/node_modules/tmdb-ts/.bun-tag-21a58013f5390430 b/.bun-tag-21a58013f5390430
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/endpoints/movies.d.ts b/dist/endpoints/movies.d.ts
index 8a62ecbd2137f24ea7d00bacca3a28a717085e90..d5ab6b598d804dae84345ede9777337fbfade12d 100644
--- a/dist/endpoints/movies.d.ts
+++ b/dist/endpoints/movies.d.ts
@@ -9,7 +9,7 @@ export interface MoviesImageSearchOptions extends LanguageOption {
 export declare class MoviesEndpoint extends BaseEndpoint {
     protected readonly accessToken: string;
     constructor(accessToken: string, customFetch?: typeof fetch);
-    details<T extends AppendToResponseMovieKey[] | undefined>(id: number, appendToResponse?: T, language?: string): Promise<AppendToResponse<MovieDetails, T, "movie">>;
+    details<T extends AppendToResponseMovieKey[] | undefined>(id: number, appendToResponse?: T, language?: string, extraParams?: Record<string, any>): Promise<AppendToResponse<MovieDetails, T, "movie">>;
     alternativeTitles(id: number): Promise<AlternativeTitles>;
     changes(id: number, options?: ChangeOption): Promise<Changes<MovieChangeValue>>;
     credits(id: number, options?: LanguageOption): Promise<Credits>;
diff --git a/dist/endpoints/movies.js b/dist/endpoints/movies.js
index 896a3784e370519972df2848655615ff61b9f258..a0c4214b1958c016cdb0e93821dd7e88c87fd47e 100644
--- a/dist/endpoints/movies.js
+++ b/dist/endpoints/movies.js
@@ -8,12 +8,13 @@ class MoviesEndpoint extends base_1.BaseEndpoint {
         super(accessToken, customFetch);
         this.accessToken = accessToken;
     }
-    async details(id, appendToResponse, language) {
+    async details(id, appendToResponse, language, extraParams = {}) {
         const options = {
             append_to_response: appendToResponse
                 ? appendToResponse.join(',')
                 : undefined,
             language: language,
+            ...extraParams
         };
         return await this.api.get(`${BASE_MOVIE}/${id}`, options);
     }
diff --git a/dist/endpoints/tv-shows.d.ts b/dist/endpoints/tv-shows.d.ts
index 1b1ed493364af5e629282b7037ad624bec3a46e1..7b6222480763fa1db2223b8fb43934aa5f07d28d 100644
--- a/dist/endpoints/tv-shows.d.ts
+++ b/dist/endpoints/tv-shows.d.ts
@@ -3,7 +3,7 @@ import { AggregateCredits, AlternativeTitles, AppendToResponse, AppendToResponse
 export declare class TvShowsEndpoint extends BaseEndpoint {
     protected readonly accessToken: string;
     constructor(accessToken: string, customFetch?: typeof fetch);
-    details<T extends AppendToResponseTvKey[] | undefined>(id: number, appendToResponse?: T, language?: string): Promise<AppendToResponse<TvShowDetails, T, "tvShow">>;
+    details<T extends AppendToResponseTvKey[] | undefined>(id: number, appendToResponse?: T, language?: string, extraParams?: Record<string, any>): Promise<AppendToResponse<TvShowDetails, T, "tvShow">>;
     alternativeTitles(id: number): Promise<AlternativeTitles>;
     changes(id: number, options?: ChangeOption): Promise<Changes<TvShowChangeValue>>;
     contentRatings(id: number): Promise<ContentRatings>;
diff --git a/dist/endpoints/tv-shows.js b/dist/endpoints/tv-shows.js
index 3174989c90b816de7dc5afa17c3bcee524d976d1..2e3a6d368a0633bcf4e140f0782ccf4248d1f708 100644
--- a/dist/endpoints/tv-shows.js
+++ b/dist/endpoints/tv-shows.js
@@ -8,12 +8,13 @@ class TvShowsEndpoint extends base_1.BaseEndpoint {
         super(accessToken, customFetch);
         this.accessToken = accessToken;
     }
-    async details(id, appendToResponse, language) {
+    async details(id, appendToResponse, language, extraParams = {}) {
         const options = {
             append_to_response: appendToResponse
                 ? appendToResponse.join(',')
                 : undefined,
             language: language,
+            ...extraParams
         };
         return await this.api.get(`${BASE_TV}/${id}`, options);
     }
