diff --git a/index.js b/index.js
index 7ece983f1dbe70a02e4a81fb58f12f10ce188065..bc1a6b2bc392e2096fd17e04fac074e7ac61cd24 100644
--- a/index.js
+++ b/index.js
@@ -176,7 +176,8 @@ module.exports.getGlobalOrigin = getGlobalOrigin
 const { CacheStorage } = require('./lib/web/cache/cachestorage')
 const { kConstruct } = require('./lib/core/symbols')
 
-module.exports.caches = new CacheStorage(kConstruct)
+// Note: this line contains code that is not bun-compatible, but it's not needed for us anyway.
+// module.exports.caches = new CacheStorage(kConstruct)
 
 const { deleteCookie, getCookies, getSetCookies, setCookie, parseCookie } = require('./lib/web/cookies')
 
